src/components/Quote.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect } from 'react'
+import ApiNinja from '../hooks/api'
+
+export default function Quote() {
+
+  const { data } = ApiNinja()
+
+  useEffect(()=>{
+    console.log(data)
```

Comment:
- [ ] Kindly remove the console.log statement, based on javascript best practices, Do not commit console.log to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website 🤗